libs/template/src/lib/core/render-aware/render-aware_creator.ts

```diff
@@ -94,7 +95,8 @@ export function createRenderAware<U>(cfg: {
           tap(cfg.templateObserver),
           renderWithLatestStrategy(strategy$)
         )
-    )
+    ),
+    publishReplay(1)
```

Comment:
We have to use publish here as we would create confusing output if new subscribers are there.
```suggestion
    publish()
```